### DISCUSSION TITLE
克隆master分支的文档如何build

### DISCUSSION THREAD

**Discussion by hanfengcan** [didi/LogicFlow#1932] [Q&A]
因为需要在内网部署文档，尝试克隆主分支后执行build之后，最终报错了。但是npm run start是可以正常打开文档的。

![image](https://github.com/user-attachments/assets/e5ca54b7-926e-4e9d-91bc-008d6ea62d1d)

**Comment by boyongjiong** [comment]
本地下载项目中，在根路径 pnpm install（理论上会执行 pnpm run build）

保险起见，你可以再在根目录运行一下 pnpm run build

然后切到 docs/site 文件夹下，pnpm run build

打包后的资源在 dist 中，部署可以参考 https://umijs.org/blog/develop-blog-using-umi#%E9%83%A8%E7%BD%B2

**Reply by hanfengcan** [reply]
build报错了，提示 :language/ 不是有效路径😂
